tests/www/test_views.py

```diff
@@ -430,7 +430,7 @@ def prepare_dagruns(self):
             state=State.RUNNING)
 
     def test_index(self):
-        with assert_queries_count(38):
+        with assert_queries_count(39):
```

Comment:
Why?

Reply:
FAB protects menu items and checks if you can click the button. Unfortunately, each item in the menu is checked in a separate database query.